app/customer_routes.py

```diff
@@ -0,0 +1,243 @@
+from re import U
```

Comment:
I like how it looks like you were trying to break up the `routes.py` file between the types of routes.  It doesn't look like you've done so however.